src/components/Tile/Tile.js

```diff
@@ -1,39 +1,21 @@
 import './Tile.css';
-import { useEffect, useRef } from 'react';
-import { useMediaTrack } from '@daily-co/daily-react-hooks';
+import { DailyVideo } from '@daily-co/daily-react';
 import Username from '../Username/Username';
-import TileVideo from '../TileVideo/TileVideo';
 
 export default function Tile({ id, isScreenShare, isLocal, isAlone }) {
   let containerCssClasses = isScreenShare ? 'tile-screenshare' : 'tile-video';
-  let audioTrack = null;
-  let audioElement = null;
-  
+
   if (isLocal) {
     containerCssClasses += ' self-view';
     if (isAlone) {
       containerCssClasses += ' alone';
     }
-  } else {
-    // Only create audio track and element if the participant is not local.
-    audioTrack = useMediaTrack(id, isScreenShare ? 'screenAudio' : 'audio');
-    audioElement = useRef(null);
-
-    useEffect(() => {
-      if (audioTrack?.state === 'playable') {
-        if (audioElement?.current) {
-          (audioElement.current.srcObject =
-            audioTrack && new MediaStream([audioTrack.persistentTrack]));
-        }
-      }
-    }, [audioTrack]);
   }
 
   return (
     <div className={containerCssClasses}>
-      <TileVideo id={id} isScreenShare={isScreenShare} />
-      {!isLocal && audioTrack && <audio autoPlay playsInline ref={audioElement} />}
+      <DailyVideo automirror sessionId={id} type={isScreenShare ? 'screenVideo' : 'video'} />
```

Comment:
To make sure I understand: Does `DailyVideo` contain both audio and video, and does it already not play audio for the local participant by default? Eg. will this work as expected if the video is disabled but the audio is not?

Reply:
If I understand correctly, DailyAudio is now loaded in `Call.js`, so if you disable video, it shouldn't affect the audio. `DailyAudio` takes care of _everyone_'s audio, and isn't tied to a tile or a participant any longer.

Reply:
Yes, what @nienkedekker said is correct. But if you want more control over each audio track, you can use `DailyAudioTrack` instead.